src/www/ui/api/Controllers/SearchController.php

```diff
@@ -89,7 +89,7 @@ public function performSearch($request, $response, $args)
     }
 
     $item = GetParm("item", PARM_INTEGER);
-    $results = GetResults($item, $filename, $uploadId, $tag, 0,
+    $results = GetResults($item, $filename, $uploadId, $tag, 0, 100,
```

Comment:
Wasn't the point of this PR is to read page size from the request?

Reply:
I have hard coded size as 100 since page no. is hard coded to be 0 as well. I can change it so that both are fetched from the request.

Reply:
Hard coding is only for the api, have added page size selection in the twig page for search.